package.json

```diff
@@ -15,9 +15,10 @@
   },
   "dependencies": {
     "adm-zip": "^0.5.5",
-    "cliff": "^0.1.10",
+    "chalk": "4.1.2",
```

Comment:
Not sure how serious it is, but the latest `5.x.x` version of `chalk` doesn't work with `require` statements anymore, so I had to use the latest `4.x.x` instead 🤔